django/db/models/indexes.py

```diff
@@ -111,6 +111,10 @@ def create_sql(self, model, schema_editor, using="", **kwargs):
                 for field_name, _ in self.fields_orders
             ]
             col_suffixes = [order[1] for order in self.fields_orders]
+            if schema_editor.connection.features.supports_index_column_ordering:
+                col_suffixes = [order[1] for order in self.fields_orders]
+            else:
+                col_suffixes = [''] * len(self.fields_orders)
```

Comment:
Most databases ignore column ordering when it's not supported, unfortunately MySQL/MariaDB with `MyISAM` storage engine raise an exception:
```
django.db.utils.OperationalError: (1178, "The storage engine for the table doesn't support descending indexes")
```